src/main/java/org/example/game/Game.java

```diff
@@ -1,85 +1,104 @@
 package org.example.game;
 
-import org.example.dto.GameStepDTO;
+import org.example.dto.GameStateDTO;
+import org.example.dto.StepForMoveDTO;
+import org.example.myExeptions.MyCustomExceptions;
 
 import java.util.Arrays;
 import java.util.Objects;
 
 public class Game {
-    private String[][] gameField;
-    private Sign sign = Sign.X;
-    private int fieldSize = 3;
+    private CellValue cellValue = CellValue.X;
+    private final int fieldSize = 3;
+    private String[][] gameField = new String[fieldSize][fieldSize];
     private boolean gameOver = false;
 
     public Game() {
     }
 
+    public GameStateDTO move(StepForMoveDTO step) throws MyCustomExceptions {
+        if (gameOver) {
+            throw new MyCustomExceptions("Game Over. Must start a new game");
+        }
+        if (isFull()) {
+            gameOver = false;
+            throw new MyCustomExceptions("Tie");
+        }
+
+
+        if (!(isInRange(step.getX(), step.getY()))) {
+            throw new MyCustomExceptions("Coordinates must be within " + fieldSize);
+        }
+        if (!(isCellNull(step.getX(), step.getY()))) {
+            throw new MyCustomExceptions("Cell " + step.getX() + " : " + step.getY() + " is not empty");
+        }
+        setStep(step);
+        GameStateDTO gameStateDTO = getGameState();
+        cellValue = cellValue.getNextSign();
+        return gameStateDTO;
+
+    }
+
+    public String[][] new_game() {
```

Comment:
Once you will have many games, you will need to get rid of this function. This logic belongs to a constructor.